src/plugins/discover/public/application/components/discover_grid/discover_grid.scss

```diff
@@ -47,6 +47,8 @@
 
 // We only truncate if the cell is not a control column.
 .euiDataGridHeader {
+  display: flex; // fixing row header in firefox
```

Comment:
First three builds of this backport failed due to **Firefox smoke test** and can't reproduced locally. For some reason sort popover had not opened.

One of the suggestion is header row position, picture below shows gap (only in firefox). So, after applying `display: flex` to a data grid in this way overwriting`display: inline-flex` everything seems ok

![8C56E89A-B8C4-41F3-A80F-EC6B0CBFB32C_4_5005_c](https://user-images.githubusercontent.com/39378793/120673585-f982b580-c49b-11eb-98fd-eceb9f166821.jpeg)

Reply:
Great debugging! Could you also apply this change in a seperate PR to master? thx!